resolwe_bio/tests/processes/test_expression.py

```diff
@@ -216,6 +216,33 @@ def test_expression_htseq(self):
         self.assertFields(expression, 'build', 'dd-05-2009')
         self.assertFields(expression, 'feature_type', 'gene')
 
+    @tag_process('htseq-count-raw')
+    def test_expression_htseq_cpm(self):
+        with self.preparation_stage():
+            inputs = {
+                'src': 'annotation dicty.gtf.gz',
+                'source': 'DICTYBASE',
+                'species': 'Dictyostelium discoideum',
+                'build': 'dd-05-2009',
+            }
+            annotation = self.run_process('upload-gtf', inputs)
+
+            bam = {
+                'src': 'reads.bam',
+                'species': 'Dictyostelium discoideum',
+                'build': 'dd-05-2009',
+            }
+            bam = self.run_process('upload-bam', bam)
+            inputs = {
+                'alignments': bam.pk,
+                'gtf': annotation.pk,
+                'stranded': 'no',
+                'id_attribute': 'transcript_id',
+            }
+        expression = self.run_process('htseq-count-raw', inputs)
+        self.assertFile(expression, 'rc', 'reads_rc.tab.gz', compression='gzip')
+        self.assertFile(expression, 'exp', 'reads_cpm.tab.gz', compression='gzip')
+
```

Comment:
The error in saving the `source` into the output field in `htseq-count-raw` was not caught by tests. Assert values of more output fields to make sure that everything works as intended.